File: spec/python/test_imports_abs.py
import unittest

from imports_abs import ImportsAbs

class TestImportsAbs(unittest.TestCase):
    def test_imports_abs(self):
        r = ImportsAbs.from_file("src/fixed_struct.bin")

        self.assertEqual(r.len.value, 80)
        self.assertEqual(len(r.body), 80)

File: spec/lua/test_imports_abs.lua
local luaunit = require("luaunit")

require("imports_abs")

TestImportsAbs = {}

function TestImportsAbs:test_import_abs()
    local r = ImportsAbs:from_file("src/fixed_struct.bin")

    luaunit.assertEquals(r.len.value, 80)
    luaunit.assertEquals(r.body:len(), 80)
end
